website/docs/configuration/environment_variables.md

```diff
@@ -37,3 +37,4 @@ TeslaMate accepts the following environment variables for runtime configuration:
 | **IMPORT_DIR**                    | The path of the directory for the import of data (e.g. TeslaFi)                                                                                                                                                                                                                                                  | ./import                      |
 | **TZ**                            | Used to establish the local time zone, e.g. to use the local time in logs. See [List of tz database time zones](https://en.wikipedia.org/wiki/List_of_tz_database_time_zones).                                                                                                                                   |                               |
 | **DEFAULT_GEOFENCE**              | The default GEOFENCE to send via GEOFENCE if car not in geofence. Overrides the default of "" which will delete any retained value.                                                                                                                                                                              | "" (no quotes)                |
+| **TESLAFI_IMPORT_VEHICLE_ID**     | The default Vehicle ID used when importing from TeslaFi. If not defined and the import source is missing that datum, 1 will be used.
```

Comment:
datum -> If it is not defined and the import source does not contain the data, 1 is used.